plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -298,13 +297,15 @@ def _get_and_save_file_units(filename, processing_function, tag, conduit, repo_i
     :param conduit:  provides access to relevant Pulp functionality
     :type  conduit:  pulp.plugins.conduits.upload.UploadConduit
 
-    :param repo_id:  id of the repo into which unit will be uploaded
-    :type  repo_id:  str
+    :param repo: The repository to import the package into
+    :type  repo: pulp.server.db.model.Repository
     """
+    repo_id = repo.repo_id
```

Comment:
From what I'm reading, I suspect this statement should not be here. If `repo` is of type `pulp.server.db.model.Repository` as the docstring says, then it should already be the correct type for the first positional param of `associate_single_unit` according to [its docustring](https://github.com/pulp/pulp/blob/0e2560e045dad843950461affad51d69c4b9f9d2/server/pulp/server/controllers/repository.py#L320). Am I reading this right?